google/cloud/spark_connect/session.py

```diff
@@ -136,14 +138,14 @@ def remote(self, url: Optional[str] = None) -> "SparkSession.Builder":
             else:
                 return self
 
-        def create(self) -> "SparkSession":
+        def create(self) -> "GoogleSparkSession":
```

Comment:
Does this have any side effect on UX? Do we still enforce this session being singleton?

Reply:
This shouldn't change the behavior except for the IDEs to now also start suggesting methods on GoogleSparkSession.